### PR TITLE
Update framework

### PR DESCRIPTION
**Pull request by peppy**
- [x] Framework bump
- [x] Test fixes for pooling changes

### CODE REVIEW

File: osu.Desktop/Program.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public static void Main(string[] args)
                 }
             }
 
-            using (DesktopGameHost host = Host.GetSuitableDesktopHost(gameName, new HostOptions { BindIPC = !tournamentClient }))
+            using (DesktopGameHost host = Host.GetSuitableDesktopHost(gameName, new HostOptions { IPCPort = !tournamentClient ? OsuGame.IPC_PORT : null }))
```
Comment (frenzibyte) [inline osu.Desktop/Program.cs:105]: Can we give debug builds a different IPC port so one can run a release version of lazer while a debug version is already running? That has been bothering me for quite some time now.

Comment (peppy) [reply osu.Desktop/Program.cs:105]: I guess yeah

Comment (bdach) [reply osu.Desktop/Program.cs:105]: > so one can run a release version of lazer while a debug version is already running

I don't ever recall needing to do that though? What's the rationale?

Comment (frenzibyte) [reply osu.Desktop/Program.cs:105]: Often times I would be on a local version of lazer and then notice something weird. Fastest way for me to check it's not my fault or it's not a regression would be to fire up a release instance and check if it happens there.

**Review by bdach — APPROVED** [review]

**Pull request merged**